runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -395,6 +384,9 @@ NNFW_STATUS nnfw_session::prepare()
 
   try
   {
+    _tracing_ctx = std::make_unique<onert::util::TracingCtx>(_subgraphs.get());
+    _compiler =
+      std::make_unique<onert::compiler::Compiler>(_subgraphs, _tracing_ctx.get(), *_coptions);
```

Comment:
@hseok-oh I think you will move creation of `TracingCtx` into `Compiler`. Then, it will have only `_compiler` constructor here.